dev/com.ibm.ws.security.client_fat/build.gradle

```diff
@@ -0,0 +1,1033 @@
+ **
+ ** SSLHttpsClient.jar
+ **
+ ******************************************************************
+ ******************************************************************/
+task SSLHttpsClient_JAR (type: Zip, dependsOn: classes) { 
+  destinationDirectory = new File(appBuildDir)
+  archiveFileName = 'SSLHttpsClient.jar'
+  
+  from (new File(projectDir, 'test-applications/SSLHttpsClient.jar/resources')) {
+    include 'META-INF/**'
+  }
+  from (new File(projectDir, 'build/classes/java/main')) {
+    include 'com/ibm/ws/security/client/fat/BasicSSLClientMain*.class'
+  }
+}
+
+/******************************************************************
+ ******************************************************************
+ **
+ ** SSLClient.ear
+ **
+ ******************************************************************
+ ******************************************************************/
+task SSLClient_EAR (type: Zip, dependsOn: SSLHttpsClient_JAR) {
+  destinationDirectory = new File(appBuildDir)
+  archiveFileName = 'SSLClient.ear'
+
+  from (new File(projectDir, 'test-applications/SSLClient.ear/resources')) {
+    include 'META-INF/**'
+  }
+  from (new File(buildDir, 'test-application')) {
+    include 'SSLHttpsClient.jar' 
+  }
+}
+
+/******************************************************************
+ ******************************************************************
+ **
+ ** BasicCalculatorClient.jar
+ **
+ ******************************************************************
+ ******************************************************************/
+task BasicCalculatorClient_JAR (type: Zip, dependsOn: classes) { 
+  destinationDirectory = new File(appBuildDir)
+  archiveFileName = 'BasicCalculatorClient.jar'
+  
+  from (new File(projectDir, 'test-applications/BasicCalculatorClient.jar/resources')) {
+    include 'META-INF/**'
+  }
+  from (new File(projectDir, 'build/classes/java/main')) {
+    include 'com/ibm/websphere/samples/technologysamples/basiccalcclient/*.class'
+    include 'com/ibm/websphere/samples/technologysamples/basiccalcclient/common/*.class'
+    include 'com/ibm/websphere/samples/technologysamples/ejb/stateless/basiccalculatorejb/BasicCalculator.class'
+    include 'com/ibm/websphere/samples/technologysamples/ejb/stateless/basiccalculatorejb/BasicCalculatorHome.class'
+  }
+}
+
+/******************************************************************
+ ******************************************************************
+ **
+ ** BasicCalculatorClient.ear
+ **
+ ******************************************************************
+ ******************************************************************/
+task BasicCalculatorClient_EAR (type: Zip, dependsOn: BasicCalculatorClient_JAR) {
+  destinationDirectory = new File(appBuildDir)
+  archiveFileName = 'BasicCalculatorClient.ear'
+
+  from (new File(projectDir, 'test-applications/BasicCalculatorClient.ear/resources')) {
+    include 'META-INF/**'
+    include 'lib/**'
```

Comment:
Not sure why these binaries are included in the application or put into github.  The APIs are provided by the liberty runtime.   I suggest removing them from github and not be included in the ear file if possible.

Reply:
When reviewing with Ryan, I realized these were here. Going to remove them.